applications/application-registry.md

```diff
@@ -0,0 +1,205 @@
+- https://www.linkedin.com/in/collin-wang-1362a51b0/
+- https://www.linkedin.com/in/djkruitbosch/
+
+
+## Development Status
+
+- No development work has started as of this submission
+- EOS Wallet+ Blue Paper
+- Grant proposal reviewed by Greymass team, pre-requisites for further development will be captured under Milstone 1
+- Grant route followed after discussion with and recommendation by Yves
+
+
+## Development Roadmap
+
+### Milestone Summary
+
+- **Total Estimated Duration:** 16 weeks 
+- **Full-Time Equivalent (FTE):**  2.4 FTE
+- **Total Costs:** 229,000 USD
+
+
+### Milestone 1 — Initiate Project & Agree on governance structure and Application Registry metadata
+
+- **Estimated duration:** 3 weeks
+- **FTE:**  0.9
+- **Costs:** 18,081 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1. | License | MIT |
+| 2. | Project Governance setup | Team setup, governance setup, stakeholder mapping complete. |
+| 3. | Governance agreement | Agreement on how the EOS Mainnet will provide governance on how Application developers can register their applications into the Application Registry. This includes a list of validators who will ensure that apps are following the defined rules. This also defines the rules that apps have to comply with, what metrics they have to support/report and how the app can be updated or deleted. Part of the governance will also consider user input based on their experience with apps. If an app is not behaving as per guidelines, it can be reported. We propose to use a 3/5 multisig with a dedicated governing body. We suggest these are the following entities: greymass, tokenpocket, ENF, ultra and rewired. Part of this deliverable we will also define the application data standards, such as title, description, category, icon, URL, app manifest, etc. |
+
+
+### Milestone 2 — Build Application Registry
+
+- **Estimated Duration:** 15 weeks
+- **FTE:**  1.2
+- **Costs:** 106,081 USD *
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | MIT |
+| 0b. | Documentation | We will provide documentation as part of the source code and a handbook on how wallet or application developers can use the Application Registry smart contract. It will contain how to build and deploy the code, how to interact with the contract, how to use EOSJS. |
+| 0c. | Testing Guide | We will provide automated Smart Contract test cases using DUNE. |
+| 1. | Application Registry (Smart Contract) | The Smart Contract will be built in such a way that it considers the governance as defined in an earlier step and will allow the Application metadata to be stored on-chain. The Smart Contract will be designed to be deployed on any EOSIO chain. |  
+
+\* Additional sub-milestones may be required to support effective cash flow for the project team
+
+
+### Milestone 3 — Build Application Registry
+
+- **Estimated Duration:** 7 weeks
```

Comment:
To what degree is this estimation affected by the discovery in Milestone 1 and any implementation decisions in Milestone 2?

Reply:
Our goal and role will be to achieve consensus on the scope at each milestone, before proceeding to the next. The scope of milestone 3 and beyond are not significantly impacted by metadata, governance, and smart contract delivery within the overall defined scope.